packages/react-components/src/templates/TagsPageBody.tsx

```diff
@@ -29,8 +34,27 @@ const MessageBody: React.FC<{ title: string; body: string }> = ({
   </main>
 );
 
+const EntityCard: React.FC<TagsPageBodyProps['results'][number]> = ({
+  ...data
+}) => {
+  if ('keywords' in data) {
+    return <SharedResearchCard {...data} />;
+  }
+
+  if ('expertiseAndResourceTags' in data) {
+    return <PeopleCard {...data} />;
+  }
+
+  return null;
```

Comment:
Could we type this against the type in the model and never returns null? What I am looking to do is make sure if a new entity is added to algolia then its either handled or excluded from the type and search response. Right now it silently fail and we would get pages of 10 showing less results which would be harder to pickup on.